docs/pages/guides/authentication/express.mdx

```diff
@@ -70,6 +88,22 @@ app.post("/api/auth", (req, res) => {
       res.status(403).end();
     });
 });
+
+```
+The `userId` used in the `authorize` function corresponds to userId used in our APIs when setting permissions (e.g. in create room).
+You can use the following example to use data from `userId` and `userInfo on the front end.
+```ts
+function Component() {
+  const self = useSelf();
+
+  // "123"
+  console.log(self.id);
+
+  // "red"
+  console.log(self.info.color);
+
+  // ...
+}
```

Comment:
```suggestion
const self = room.getSelf();

// "123"
console.log(self.id);

// "red"
console.log(self.info.color);
```

Express could be any framework, so using `@liveblocks/client` instead of `@liveblocks/react` here